bepdf/beos/PreferencesWindow.cpp

```diff
@@ -59,13 +59,14 @@ void PreferencesWindow::UpdateWorkspace() {
 }
 
 void PreferencesWindow::BuildWorkspaceMenu(BMenu *m) {
-	m->AddItem(new BMenuItem(B_TRANSLATE("current"), new BMessage(WORKSPACE_CHANGED)));
+	m->AddItem(new BMenuItem(B_TRANSLATE("Current"), new BMessage(WORKSPACE_CHANGED)));
 	m->AddSeparatorItem();
 	int n = count_workspaces();
+
+    BString buffer;
 	for (int i = 1; i <= n; i ++) {
-		char buffer[10];
-		sprintf(buffer, "%d", i);
-		m->AddItem(new BMenuItem(buffer, new BMessage(WORKSPACE_CHANGED)));
+        buffer.SetToFormat("%d", i);
```

Comment:
```suggestion
		buffer.SetToFormat("%d", i);
```

Reply:
Oh, I got it. Tabs versus spaces. But shouldn´t this buffer... get aligned with the previous ( m-> ) line ?

Reply:
Sure.